src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends AbstractFigure {
+    private double width;
+    private double height;
+
+    public Rectangle(String color, double width, double height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double calculatedArea() {
+        return width * height;
+    }
```

Comment:
There is a typo in the method name 'calculatedArea'. It should be 'calculateArea' to follow standard naming conventions and to match the expected method name in the Figure interface.